pkg/test/masteruserrecord/master_user_record.go

```diff
@@ -74,36 +159,89 @@ func ModifyUaInMur(mur *toolchainv1alpha1.MasterUserRecord, targetCluster string
 }
 
 func StatusCondition(con toolchainv1alpha1.Condition) MurModifier {
-	return func(mur *toolchainv1alpha1.MasterUserRecord) {
+	return func(mur *toolchainv1alpha1.MasterUserRecord) error {
 		mur.Status.Conditions, _ = condition.AddOrUpdateStatusConditions(mur.Status.Conditions, con)
+		return nil
 	}
 }
 
 func MetaNamespace(namespace string) MurModifier {
-	return func(mur *toolchainv1alpha1.MasterUserRecord) {
+	return func(mur *toolchainv1alpha1.MasterUserRecord) error {
 		mur.Namespace = namespace
+		return nil
 	}
 }
 
 func Finalizer(finalizer string) MurModifier {
-	return func(mur *toolchainv1alpha1.MasterUserRecord) {
+	return func(mur *toolchainv1alpha1.MasterUserRecord) error {
 		mur.Finalizers = append(mur.Finalizers, finalizer)
+		return nil
 	}
 }
 
 func TargetCluster(targetCluster string) MurModifier {
-	return func(mur *toolchainv1alpha1.MasterUserRecord) {
+	return func(mur *toolchainv1alpha1.MasterUserRecord) error {
 		for i := range mur.Spec.UserAccounts {
 			mur.Spec.UserAccounts[i].TargetCluster = targetCluster
 		}
+		return nil
+	}
+}
+
+func Account(cluster string, tier toolchainv1alpha1.NSTemplateTier) MurModifier {
+
+	return func(mur *toolchainv1alpha1.MasterUserRecord) error {
+		// set the user account
+		templates := nstemplateSetFromTier(tier)
+		mur.Spec.UserAccounts = append(mur.Spec.UserAccounts, toolchainv1alpha1.UserAccountEmbedded{
+			TargetCluster: "member-cluster",
+			Spec: toolchainv1alpha1.UserAccountSpecEmbedded{
+				UserAccountSpecBase: toolchainv1alpha1.UserAccountSpecBase{
+					NSTemplateSet: templates,
+				},
+			},
+		})
+		// set the labels for the tier templates in use
+		hash, err := computeTemplateRefsHash(tier)
+		if err != nil {
+			return err
+		}
```

Comment:
I don't understand what's wrong here: I'm configuring the MasterUserRecord with an account which uses an outdated tier. When the tier will be updated, I can select this particular MasterUserRecord because its "hash" label value changed. What's wrong with that?

Reply:
never mind - I didn't have the context of the usage in host-operator. I didn't realize that you would add a different tier than is used in the first UserAccount